scripts/superdesk-highlights/highlights.js

```diff
@@ -560,7 +563,7 @@
                             'button',
                             {disabled: isMarked, onClick: this.markHighlight},
                             React.createElement('i', {className: 'icon-star'}),
-                            highlight.name
+                            highlight.desks.length ?  highlight.name : highlight.name + gettext(' (Global)')
```

Comment:
I think this logic can be moved to highlight service instead of repeating in few places...